src/pybind/mgr/dashboard/tox.ini

```diff
@@ -176,6 +177,20 @@ commands =
      python3 -m dashboard.controllers.docs {env:OPENAPI_FILE_TMP:{env:OPENAPI_FILE}}
      check: diff {env:OPENAPI_FILE} {env:OPENAPI_FILE_TMP}
 
+[testenv:grafonnet-{check,fix}]
+basepython = python3
+whitelist_externals =
+    jsonnet
+    bash
+description =
+    check: Ensure that auto-generated grafana dashboard files matches the current version
+    fix: generate dashboard json files from jsonnet file with latest changes
+deps =
+    {[base]deps}
```

Comment:
Do we really need all the python deps for this? Maybe it makes more sense to install the only python dep we need here from a separate `requirements-grafonnet.txt`?

Reply:
Sure